.azure-pipelines/js-mgmt-ci.yml

```diff
@@ -53,3 +53,27 @@ jobs:
       - task: PublishBuildArtifacts@1
         inputs:
           pathtoPublish: $(Build.SourcesDirectory)/drop
+
+  - job: 'Analyze'
+
+    pool:
+      vmImage: 'Ubuntu 16.04'
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '$(NodeVersion)'
+        displayName: 'Install Node.js $(NodeVersion)'
+
+      - task: Npm@1
+        displayName: 'npm install'
+        inputs:
```

Comment:
While this task does work since the default is `command: install`, I think this is very non-obvious so I would add `command: install`.

Reply:
done